duckduckgo/iOS
# NetP: Remove port from server address

Task/Issue URL: https://app.asana.com/0/0/1206056874665890/f

**Description**:

During the NetP iOS ship review, there was feedback that the server IP address labels should not feature the port. So now, rather than sending the host + port as the server address, I’m just sending the host.

**Steps to test this PR**:
1. Launch NetP
2. Check the status view and make sure the IP Address does not include the port



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
